package-lock.json

```diff
@@ -1,5135 +1,13 @@
 {
   "name": "cardano-transaction-lib",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Do you know why this file reduced its size that much?

Reply:
It's a different lockfile version I think

Reply:
@klntsky It's a correctly re-generated `package-lock.json`. It looks like I generated it the wrong way before, sorry for that.